presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -485,67 +489,37 @@ public boolean isAggregationFunction(QualifiedName name)
 
     public Signature resolveFunction(QualifiedName name, List<TypeSignature> parameterTypes, boolean approximate)
     {
-        List<SqlFunction> candidates = functions.get(name).stream()
+        List<SqlFunction> allCandidates = functions.get(name).stream()
                 .filter(function -> function.getSignature().getKind() == SCALAR || (function.getSignature().getKind() == APPROXIMATE_AGGREGATE) == approximate)
                 .collect(toImmutableList());
 
+        List<SqlFunction> candidatesWithExplicitlySpecifiedParameters = allCandidates.stream()
```

Comment:
Is `exactCandidates` ok? AFAIU we use this name in other places in the same meaning.